consensus/core/src/storage/rocksdb_store.rs

```diff
@@ -38,7 +35,7 @@ pub(crate) struct RocksDBStore {
     /// TODO: batch multiple votes into a single row.
     commit_votes: DBMap<(CommitIndex, CommitDigest, BlockRef), ()>,
     /// Stores the latest values of a few properties.
-    commit_info: DBMap<(CommitIndex, CommitDigest), CommitInfo>,
+    commit_info: DBMap<CommitRange, CommitInfo>,
```

Comment:
What is the reason this has to be changed to CommitRange?

Reply:
The idea was that we will only store commit_info (scores + last commit rounds) on the same duration as leader schedule change which is every 300 commits. So we store this info keyed on the `CommitRange` and we expect to retrieve/recover it based on these ranges as well

Reply:
I believe it is still better to store `(CommitIndex, CommitDigest)` of the last commit in the leader schedule as the key. Since leader schedules are contiguous, storing the start round seems unnecessary. Also technically `CommitInfo` is accumulated over all past rounds.

Reply:
I see the argument @arun-koshy , it's just that the current query patterns they seem to not really have the need to query in ranges. I am good with either but slight preference on having this the previous way

Reply:
Agreed, made the change to not use CommitRange